lib/project_types/script/layers/infrastructure/assemblyscript_task_runner.rb

```diff
@@ -37,6 +37,20 @@ def dependencies_installed?
           ctx.dir_exist?("node_modules")
         end
 
+        def check_if_safe_to_push!
```

Comment:
Yep, that's fair. I originally wrote this in a different part of the infra code that was more push oriented but realized it could be modelled as a task instead. So renaming makes sense.